### ISSUE TITLE
xml2csv-standalone-v0.2.jar does not take -i argument

### ISSUE THREAD

**Issue by letsflykite** [andybrodie/xml2csv#7]
I downloaded the xml2csv-standalone-v0.2.jar. It does not take -i

java -jar xml2csv-standalone-v0.2.jar -c PeopleConfig.xml -i input -o output
Invalid arguments specified: Unrecognized option: -i
usage: java.exe -jar xml2csv-standalone.jar [-a] -c <arg> [-h] [-o <arg>] [-v] [-w]
xml2csv v0.2.  Converts XML files in to CSV files using a user-defined set of rules.
-a,--append-output           If specified, all output will be appended to any existing output files.  If an existing file is appended to
                             then field names will not be output.
-c,--configuration-file <arg>A single file containing the configuration to use.
-h,--help                    Show help on using xml2csv and terminate.
-o,--output-directory <arg>  The directory to which the output CSV files will be written.  If not specified, current working directory will
                             be used.  Directory must exist and be writeable.
-v,--version                 Show version information and terminate.
-w,--preserve-whitespace     If specified then whitespace will not be removed from the start and end of output fields.

**Comment by andybrodie** [comment]
Hi Letsflykite,

Thanks for trying out xml2csv and getting in touch.  My apologies for your issue, I updated xml2csv a while ago to not use `-i` for specifying the input files or directories.  If you omit the `-i` completely then it should work fine.

I've updated the Wiki to remove this error from the documentation.

So, if you type:

`java -jar xml2csv-standalone-v0.2.jar -c PeopleConfig.xml input -o output`

replacing `input` and `output` with your input file/directory and the output directory, respectively, then it should work fine.

If you've cloned the Git repository, then something like this, from the project root directory, should work fine:

`java.exe -jar dist\xml2csv-standalone-v0.2.jar -c testdata\PeopleConfig.xml -o . testdata\People.xml`

Thanks for your feedback and pointing out a bug!
